### PR TITLE
[DNA page ] implement enhanced cards blcok

### LINKED ISSUE #12: [DNA page] Enhanced cards block
- [ ] Mobile version
- [ ] Desktop version

### PR DESCRIPTION
**Pull request by FentPams**
## Working in progress

Fix #12 

Test URLs:
- Before: https://main--com--ancestry.aem.live/
- After: https://dna-hero--com--ancestry.aem.page/drafts/xfeng/dna-4

**Screenshot for desktop**
Screenshot 2024-10-07 at 8 04 43 AM

_popup window_
Screenshot 2024-10-07 at 8 04 50 AM

_dialogue overlay_
Screenshot 2024-10-07 at 8 05 00 AM

**Screenshot for mobile**
Screenshot 2024-10-07 at 3 16 23 PM

Screenshot 2024-10-07 at 3 16 58 PM

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -236,10 +347,724 @@
...
+    display: none;
+  }
+
+  .section.dna.cards-container .cards-card-body .product .button-container a {
+    padding: 0.6rem 0;
+    width: 60%;
+  }
+}
+  
+@media(max-width:767px) {
```
Comment (ujjwal5) [inline blocks/cards/cards.css:911]: let's stick to 900px breakpoint as used elsewhere.

File: icons/dna-bg-worldmap.jpeg
Comment (ujjwal5) [inline icons/dna-bg-worldmap.jpeg:1]: are we using background as an icon or are we asking authors to have it as part of content ?

Comment (FentPams) [reply icons/dna-bg-worldmap.jpeg:1]: I'll delete this image, I already took your method as adding background image in the section metadata :)

**Comment by ujjwal5** [comment]
@FentPams Also, can you take latest pull from main branch. It's currently showing this branch to be out-of-date.

**Comment by FentPams** [comment]
> @FentPams Also, can you take latest pull from main branch. It's currently showing this branch to be out-of-date.

I just

**Comment by FentPams** [comment]
> @FentPams Also, can you take latest pull from main branch. It's currently showing this branch to be out-of-date.

updated :)

**Review by ujjwal5 — APPROVED** [review]

**Pull request merged**